symfony/framework-bundle/5.4/manifest.json

```diff
@@ -9,11 +9,13 @@
     },
     "composer-scripts": {
         "cache:clear": "symfony-cmd",
-        "assets:install %PUBLIC_DIR%": "symfony-cmd"
+        "assets:install %PUBLIC_DIR%": "symfony-cmd",
+        "secrets:generate-keys": "symfony-cmd",
+        "secrets:set APP_SECRET --random": "symfony-cmd",
+        "secrets:decrypt-to-local --force": "symfony-cmd"
```

Comment:
this should only be done for performance on production. it's nothing you'd always want to run.

Reply:
oki lets remove it